app/components/TeamSection/index.tsx

```diff
@@ -27,45 +29,107 @@ type TProps = {
 
 const TeamSection = () => {
   const { isMobile } = useWindowSize();
-  const { aboutUs } = GetQueries();
+  const { aboutUs, employee } = GetQueries();
   const data = useContentfulData<TProps>("aboutUsCollection", aboutUs);
+  const employees = useContentfulData<TProps>("employeeCollection", employee);
   const isValidData = data?.items && data.items.length > 0;
-  const teamSEOSrc = isValidData ? data.items[0].image.url : undefined;
+  const isValidEmployees = employees?.items && employees.items.length > 0;
+  const sortedEmployees = isValidEmployees && employees.items.sort((a, b) => b.order - a.order);
 
   return (
     <MainContainer>
       {
-        isValidData && teamSEOSrc &&
+        isValidData &&
         <section id="team" className={styles.component}>
           <H2 className={styles.title}>{data.items[0].title}</H2>
           <P className={styles.subtitle}>{data.items[0].firstSubtitle}</P>
-          <div className={styles.teamCard}>
-            <div className={styles.imageWrapper}>
-              <Image className={styles.image} src={teamSEOSrc} alt="TeamSEO" fill />
-            </div>
-            <div className={styles.teamTitle}>
-              <div className={styles.socialWrapper}>
-                <H3>{data.items[0].name}</H3>
-                <a className={styles.socialLink} href="https://www.linkedin.com/in/julian-verocai-30b1a582/" target="_blank">
-                  <Image className={styles.social} src={linkedinSrc} alt="Linkedin" />
-                </a>
-              </div>
-              <P>{data.items[0].specialty}</P>
-            </div>
-          </div>
           {
             isMobile ?
-              <div className={styles.textContainer}>
-                <P>{data.items[0].secondSubtitle}</P>
-                <P>{data.items[0].thirdSubtitle}</P>
-                <P>{data.items[0].fourthSubtitle}</P>
-              </div>
+              sortedEmployees &&
```

Comment:
please read this principle and refactor this new code https://en.wikipedia.org/wiki/Don%27t_repeat_yourself

Reply:
Refactored this code